internal/services/mssql/mssql_server_resource_test.go

```diff
@@ -762,6 +784,72 @@ resource "azurerm_mssql_server" "test" {
 `, data.RandomInteger, data.Locations.Primary)
 }
 
+func (MsSqlServerResource) password(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+provider "azuread" {}
+
+data "azurerm_client_config" "test" {}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-mssql-%[1]d"
+  location = "%[2]s"
+}
+
+resource "azurerm_mssql_server" "test" {
+  name                = "acctestsqlserver%[1]d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = azurerm_resource_group.test.location
+  version             = "12.0"
+
+  administrator_login          = "missadministrator"
+  administrator_login_password = "thisIsKat11"
+
+  azuread_administrator {
+    login_username              = "AzureAD Admin2"
+    object_id                   = data.azurerm_client_config.test.object_id
+    azuread_authentication_only = true
+  }
+}
+`, data.RandomInteger, data.Locations.Primary)
+}
+
+func (MsSqlServerResource) passwordUpdate(data acceptance.TestData) string {
```

Comment:
Similarly would rename to something like `mssqServerResourceWithAdminLogin`

Reply:
Thanks, changed to `aadAdminWithAADAuthOnlyWithPassword`.